Library of Congress ISBN.js

```diff
@@ -8,7 +8,7 @@
 	"priority": 98,
 	"inRepository": true,
 	"translatorType": 8,
-	"browserSupport": "gcsb",
+	"browserSupport": "gcsibv",
```

Comment:
True. I will bump the timestamps for the translators with extended `browserSupport`.